exchanges/exchanges_test.go

```diff
@@ -27,157 +23,6 @@ func enableTestLog() {
 	}
 }
 
-func makeKillSwitch() chan os.Signal {
-	killSwitch := make(chan os.Signal, 1)
-	signal.Notify(killSwitch, os.Interrupt)
-	return killSwitch
-}
-
-func testExchanges(asSlave, quickTest bool, t *testing.T) {
-	enableTestLog()
-
-	// Skip this test during automated testing.
-	if os.Getenv("GORACE") != "" {
```

Comment:
This was a sloppy way to skip this test during CI testing, but it's not compatible with the changes in #1347.

Reply:
Is it because the test takes too much time that you wanted to skip it?  We can leverage the `-short` switch.  https://golang.org/pkg/testing/#hdr-Skipping e.g.

```go
if !testing.Short() {
    // do long tests
...
}
```

Reply:
Or use `t.Skip` above I suppose.

Reply:
Yeah. All the live tests take a long time. I put a tag at the top of the _live file, so you have to enable it e.g. `go test -tags-livexc`.

Reply:
Oh, perfect.